qc_utils.py

```diff
@@ -45,6 +45,8 @@
             "o" : "Odd Cluster",
             "p" : "Pressure",
             "w" : "Winds",
+            "x" : "Excess streak proportion",
+            "y" : "Repeated Day streaks"
             }
```

Comment:
Is the 'humidity' one supersaturation or repeated streaks of 0 DPD?

Reply:
Both are captured by the "h" flag rather than giving these two issues separate flag characters.  Happy to discuss whether that is something to address.